### PR TITLE
Accept multiple spaces before comment in selector-list-comma-newline-after rule

### LINKED ISSUE #2912: Fix false positives for shared-line comments separated by more than once space in selector-list-comma-newline-after
With default settings rule `selector-list-comma-newline-after` complaints about following CSS:

```
.myClass,  // this is comment
.myOtherClass {
  ...
}
```

Error: `Expected newline after "," `

But does not complain when there is a single space before the comment. I might attempt to create a PR for this, if you agree with me, that it needs fixing...

### PR DESCRIPTION
**Pull request by pkuczynski**
Fixes #2912

### CODE REVIEW

File: lib/rules/selector-list-comma-newline-after/index.js [hunk 1]
```diff
@@ -45,20 +45,22 @@ const rule = function(expectation) {
...
+          );
 
           // If there's a // comment, that means there has to be a newline
           // ending the comment so we're fine
-          if (nextThreeChars === " //") {
+          if (nextChars.match(/^\s+\/\//)) {
             return;
           }
 
           // If there is a space and then a comment begins, look for the newline
```
Comment (ntwb) [inline lib/rules/selector-list-comma-newline-after/index.js]: Can this comment be updated to reflect the new functionality of the code please

How about: `If there are spaces and then a comment begins, look for the newline`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 -   Added: `length-zero-no-unit` autofix ([#2861](https://github.com/stylelint/stylelint/issues/2861)).
 -   Added: `selector-max-specificity` support for level 4 evaluation context pseudo-classes ([#2857](https://github.com/stylelint/stylelint/issues/2857)).
+-   Fixed: aAccept multiple spaces before comments in `selector-list-comma-newline-after` rule ([#2912](https://github.com/stylelint/stylelint/issues/2912)).
```
Comment (ntwb) [inline CHANGELOG.md]: ~Typo here `Accept`, not `aAccept` ;)~

Comment (ntwb) [reply CHANGELOG.md]: Actually, can you remove this change to CHANGELOG.md entirely from this PR please.

We update CHANGELOG.md per the docs here https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/pull-requests.md after the PR has been merged, it helps us avoid CHANGELOG.md merge conflicts from multiple PRs :)

**Comment by jeddy3** [comment]
@pkuczynski Thanks for this, especially the comprehensive tests. If you can make @ntwb changes, then I believe this will be good to go.

File: CHANGELOG.md (hunk 2 above)
Comment (pkuczynski) [reply CHANGELOG.md]: Sure I can. In `rvm` project we actually enforce it the other way around asking people to provide CHANGELOG entries :)

**Comment by pkuczynski** [comment]
@jeddy3 done! :)

File: CHANGELOG.md (hunk 2 above)
Comment (jeddy3) [reply CHANGELOG.md]: We did it that way around originally, but we constantly got merge conflicts in our PRs.

**Comment by pkuczynski** [comment]
Yeah, us too. But then very easy to solve using gihtub ui :) At least that's how we deal with it. But that does not matter I guess :)

**Comment by jeddy3** [comment]
> -   Fixed: `selector-list-comma-newline-after` false positives for shared-line comments separated by more than once space ([#2915](https://github.com/stylelint/stylelint/issues/2915)).

**Comment by jeddy3** [comment]
@pkuczynski Thanks again for your first-time contribution. Your fix was just released in `8.2.0`.

**Comment by pkuczynski** [comment]
Glad to hear that! I will have a look at my other PR tonight...

**Pull request merged**